app/src/main/res/layout/activity_session_feedback.xml

```diff
@@ -23,234 +23,295 @@
                 app:popupTheme="@style/ThemeOverlay.AppCompat.Light" />
```

Comment:
I know what you mean. But I guess layout nest is not big matter because I checked performance of view rendering.
http://qiita.com/konifar/items/3c75f1fab0e0a10ebb14

As you said, it is only way to add another LinearLayout or RelativeLayout. But I think it's OK.